### DISCUSSION TITLE
Does shadowsocks-android v5.2.6 support goss2?

### DISCUSSION THREAD

**Discussion by gpmn** [shadowsocks/go-shadowsocks2#221] [Q&A] [answered]
https://github.com/shadowsocks/shadowsocks-android/releases/tag/v5.2.6
I use this client, but it do not support AEAD_CHACHA20_POLY1305 which go-ss2 used.
while it support CHACHA20_IETF_POLY1305 ,is this same as AEAD_CHACHA20_POLY1305?
and which android client could support go-shadowsocks2?

**Comment by dev4u** [comment, marked as answer]
YES.
